runtime/jcl/common/mgmtthread.c

```diff
@@ -1114,39 +1103,28 @@ handlerMonitorWaited(J9HookInterface** hook, UDATA eventNum, void* eventData, vo
  * These classes should be loaded only if ThreadMXBean is used.
  * @pre must not have VM access
  * @todo When should these references be deleted?
- * @param[in] env
- * @return error status
- * @retval JNI_OK success
- * @retval JNI_ENOMEM OutOfMemoryError is set
- * @retval JNI_ERR FindClass() or GetMethodID() failed, an exception is set
+ * @param[in] env The JNI env
+ * @param[in] jcls The Class (static method)
  */
-static jint
-initIDCache(JNIEnv *env)
+void JNICALL
+Java_openj9_management_internal_IDCacheInitializer_initIDCache(JNIEnv *env, jclass jcls)
 {
-	jclass oom;
-	jclass cls;
-	jclass gcls;
-	jmethodID mid;
-	jint err = JNI_OK;
+	jclass oom = NULL;
+	jclass cls = NULL;
+	jclass gcls = NULL;
+	jmethodID mid = NULL;
 	
-	/* isNativeMethod is the last cache member to be set */
-	if (JCL_CACHE_GET(env, MID_java_lang_StackTraceElement_isNativeMethod) != NULL) 
-		return JNI_OK;
-
 	/* cache OutOfMemoryError */
 	oom = (*env)->FindClass(env, "java/lang/OutOfMemoryError");
 	if (!oom) {
-		err = JNI_ERR;
 		goto initIDCache_fail;
 	}
 	
 	cls = (*env)->FindClass(env, "openj9/management/internal/ThreadInfoBase");
 	if (!cls) {
-		err = JNI_ERR;
 		goto initIDCache_fail;
 	}
 	if (!(gcls = (*env)->NewGlobalRef(env, cls))) {
-		err = JNI_ENOMEM;
 		goto initIDCache_fail;
 	}
 	(*env)->DeleteLocalRef(env, cls);
```

Comment:
Please remove all of these from this function.

Reply:
done!